src/utils/env.ts

```diff
@@ -0,0 +1,15 @@
+export function isDevelopmentEnv(): boolean {
```

Comment:
Isn't this file more suitable for this? https://github.com/cowprotocol/cowswap/blob/27010ba5c351900d24d04f08f6441ade0d3acc77/src/custom/utils/environments.ts#L15

Reply:
Probably, I have removed used of this